client/src/components/BlockEditor/BlockEditorEditing.tsx

```diff
@@ -123,6 +136,63 @@ export class BlockEditorEditingPresentational extends React.Component<
     }
   }
 
+  public normalizeChangeWrtTextNodeSpacing(c: Change) {
```

Comment:
`BlockEditorEditing.tsx` is getting pretty big. Consider abstracting out parts. Text normalization seems like a natural candidate.

Reply:
https://github.com/oughtinc/mosaic/pull/145/commits/a245a2c623007051c9b6d4c0a0bd4c4e83fc26da